src/App.jsx

```diff
@@ -14,70 +17,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        { value ? (
+          <span>
+            { `${value} is selected` }
+          </span>
+        ) : (
+          <span>No goods selected</span>
+        )
+        }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+        { value && (
+          <>
             <button
-              data-cy="AddButton"
+              onClick={ () => {
+                setValue(null);
+              } }
```

Comment:
move to own handler function
```
const reset = () => {
  setSelectedGood(null);
}
```

```suggestion
              onClick={reset}
```